### PR TITLE
Cjs/feature/new briefings count

### PR DESCRIPTION
**Pull request by robojack**
On the initial app launch, we save the most recent briefing in local storage. Upon subsequent app launches, if there are newer briefings then the one we have stored, we calculate the number of new briefings and display it as a badge on the Recent tab.

When the user visits the Recent tab, we update our local storage with the newest briefing but we DO NOT reset the badge. However, when the user relaunches the app, the badge count will be derived from the previous launch.

When the user pulls to refresh in the Recent tab, the badge is immediately reset.

### CODE REVIEW

File: app/components/Tabs/TabBar.js [hunk 1]
```diff
@@ -17,8 +30,16 @@ const TabBar = () => {
...
             style={[styles.tab, active ? styles.active : null]}
             testID={`tab-bar-touchable-${t}`}
-            onPress={() => setCurrentTab(t)}>
+            onPress={() => onPress(t)}>
             <TiText style={[active ? styles.activeText : null]}>{t}</TiText>
+
+            {t === badgeTab && badgeCount > 0 && (
+              <View style={styles.badge}>
+                <TiText allowFontScaling={false} style={styles.badgeCount}>
+                  {badgeCount > 10 ? '+10' : badgeCount}
```
Comment (bolandrm) [inline app/components/Tabs/TabBar.js]: Should this be `10+` instead of `+10`?

**Comment by bolandrm** [comment]
@robojack This looks good!  I'm curious why you chose to go with store the GUID of the most recent instead of the date.  It seems to me like using the date could have simplified some things  (e.g. searching through the list to find a specific GUID).

File: app/components/Tabs/TabBar.js (hunk 1 above)
Comment (robojack) [reply app/components/Tabs/TabBar.js]: good catch, it should be `10+`

**Comment by robojack** [comment]
@bolandrm good question. I probably had my dev blinders on and was not thinking through this from a purely chronological perspective. It definitely makes sense to treat it as such. Update coming shortly.

**Pull request merged**